src/contrib/cluster/Akka.Cluster.Tools/Client/ClusterClientSettings.cs

```diff
@@ -19,7 +19,12 @@ public sealed class ClusterClientSettings
     {
         public static ClusterClientSettings Create(ActorSystem system)
         {
+            system.Settings.InjectTopLevelFallback(ClusterClientReceptionist.DefaultConfig());
```

Comment:
Yes, I agree, but ClusterClient, ClusterSingletonManager and ClusterSingletonProxy don't have GetExtension method